src/app/components/user/pin.service.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+(function () {
+  'use strict';
+
+  angular.module('janusHangouts')
+    .service('PinService', PinService);
+
+  function PinService() {
```

Comment:
Why do you need a PinService at all? I think this information should belong to user.